File: spec/code_samples/C#/webhooks@{id}/patch.cs
var api = CheckoutApi.Create("your secret key");

// Original webhook
var webhook = new Webhook()
{
  Url = "https://example.com/webhooks",
  EventTypes = new List<string>
    {
        "payment_pending",
        "payment_captured"
    },
  Headers = new Dictionary<string, string>
    {
        { "Authorization", "1234" }
    }
};
var webhookRegistrationResponse = await Api.Webhooks.RegisterWebhookAsync(new RegisterWebhookRequest(webhook));

// Partially update
webhook.Url += "/partially/updated";
webhook.Headers = null;
var webhookPartialUpdateResponse = await Api.Webhooks.PartiallyUpdateWebhookAsync(
                                                        webhookRegistrationResponse.Id,
                                                        new PartialUpdateWebhookRequest(webhook)
                                                      );

File: abc_spec/code_samples/Node/webhooks@{id}/patch.js
import { Checkout } from 'checkout-sdk-node';

const cko = new Checkout('sk_XXXX');

const webhook = await cko.webhooks.partiallyUpdateWebhook('wh_ahun3lg7bf4e3lohbhni65335u', {
	url: 'https://example.com/webhooks/updated',
});
